inscriptions/ISic000525.xml

```diff
@@ -190,14 +195,23 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#GC">
+                <ab>
+                    <lb n="1"/><name>Pituaniae</name>
+                    <lb n="2"/><expan><abbr>c</abbr><ex>larissimae</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>f</abbr><ex>iliae</ex></expan> <g ref="#interpunct">·</g> <name>Vettuleni</name>
+                    <lb n="3"/><name>Quadrati</name> <g ref="#interpunct">·</g> <expan><abbr>q</abbr><ex>uiescit</ex></expan>
```

Comment:
EDCS suggests q(uaestoris), which seems more plausible to me, so that we have then 'to Pituania, daughter of Gaius Vettulenus Quadratus, quaestor'